### ISSUE TITLE
hidden more button

### ISSUE THREAD

**Issue by hchiam** [hchiam/slides#105]
# Problem/Need:

To keep the main interface simple, while enabling users to use the other functions without resorting to the console, and while staying as accessible (a11y) as possible. Some other ideas for extra functionality should be out of sight for non-advanced users. 

# Suggestions:

- hidden "More" / "+" button that's only visible once you tab past the last button

**Comment by hchiam** [comment]
save/reuse buttons (#89)
slides titler button (#104)
